### ISSUE TITLE
Repeated opening/changing standards drop down shows errors on analytics code

### ISSUE THREAD

**Issue by tmcconechy** [squizlabs/HTML_CodeSniffer#148]
On the current bookmarklet (live)

   - click to open - its on WCAGAA and seems correct
   - select WCAGAAA - it reports and seems correct
   - select WCAGAA again
   - select WCAGAAA again

Each time the number of errors increases. Showing errors in some markup google analytical tags....

**Comment by ironikart** [comment]
Unable to replicate this with the steps given. Have tried with a few variations of google analytics tags. Is it possible to get a more specific example of the page where this is occuring?

**Comment by tmcconechy** [comment]
Seems ok now. I think the linked issue must have also fixed this. Thanks
